CHANGELOG.MD

```diff
@@ -4,9 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0-beta1] - September 27th, 2019
 Changes that have landed but are not yet released.
```

Comment:
You should keep the `Unreleased` section and create a new heading for this release